src/missing.c

```diff
@@ -260,6 +260,138 @@ r_ssize list_col_equal_na(r_obj* x,
 
 // -----------------------------------------------------------------------------
 
+// [[ register() ]]
+r_obj* ffi_vec_any_missing(r_obj* x) {
+  return r_lgl(vec_any_missing(x));
+}
+
+// [[ include("missing.h") ]]
```

Comment:
By the way I think we no longer need include statements, since we decided that any non static functions must be exported from the companion header file.

Reply:
> any non static functions must be exported from the companion header file.

I didn't know/remember we decided that but that works for me 😆

Reply:
And exported functions get an `ffi_` prefix so we don't need `register()` either? (im going to remove it)

Reply:
That makes sense.